scripts/qa_crowbarsetup.sh

```diff
@@ -4451,11 +4465,7 @@ function onadmin_crowbarrestore()
             with_upgrade)
                 # restore after upgrade has different workflow (missing APIs) than
                 #   a restore from a backup of the same cloud release
-                local http_code=`curl -X POST -F "file=@/tmp/$btarball" -s -o upgrade-start.txt -w '%{http_code}' $crowbar_api/installer/upgrade/start.json`
```

Comment:
the other thing/difference is we didn't use authentification here, and now we are (its also not necessarily needed, as we are already on the admin node), but that shouldn't make any difference